aio/content/examples/cli-builder/src/my-builder.ts

```diff
@@ -3,44 +3,42 @@
 import { BuilderContext, BuilderOutput, createBuilder } from '@angular-devkit/architect';
 import { JsonObject } from '@angular-devkit/core';
 // #enddocregion builder-skeleton
-import * as childProcess from 'child_process';
+import { promises as fs } from 'fs';
 // #docregion builder-skeleton
 
 interface Options extends JsonObject {
-  command: string;
-  args: string[];
+  source: string;
+  destination: string;
 }
 
-export default createBuilder(commandBuilder);
+export default createBuilder(copyFileBuilder);
 
-function commandBuilder(
+async function copyFileBuilder(
   options: Options,
   context: BuilderContext,
-  ): Promise<BuilderOutput> {
-    // #enddocregion builder, builder-skeleton, handling-output
-    // #docregion report-status
-    context.reportStatus(`Executing "${options.command}"...`);
-    // #docregion builder, handling-output
-    const child = childProcess.spawn(options.command, options.args);
-    // #enddocregion builder, report-status
-
-    child.stdout.on('data', data => {
-      context.logger.info(data.toString());
-    });
-    child.stderr.on('data', data => {
-      context.logger.error(data.toString());
-    });
-
+): Promise<BuilderOutput> {
+  // #enddocregion builder, builder-skeleton, handling-output
+  // #docregion report-status
+  context.reportStatus(`Copying ${options.source} to ${options.destination}.`);
+  // #docregion builder, handling-output
+  try {
+    await fs.copyFile(options.source, options.destination);
+  } catch (err) {
+    // #enddocregion builder
+    context.logger.error(`Failed to copy file.`);
```

Comment:
```suggestion
    context.logger.error('Failed to copy file.');
```

Also, is the error message surfaced somewhere? Would it be helpful to include it here?

Reply:
The error gets returned and architect shows it to the user, this is just providing some additional context for the following error message.